client/src/components/TripChecklist.vue

```diff
@@ -6,15 +6,25 @@
     <table v-else class="table-fixed text-sm text-left text-gray-500 rounded-md">
       <thead class="text-lg text-gray-700 bg-gray-200">
         <tr>
-          <th class="pr-0"></th>
-          <th class="px-0 py-3 w-full">Product</th>
+          <th class="px-2 py-3 w-full" colspan="2">Product</th>
           <th class="px-1 md:px-6 py-3 whitespace-nowrap">SupplyTarget</th>
           <th class="px-1 md:px-6 py-3 whitespace-nowrap">Qty</th>
           <th class="px-2 md:px-6 py-3 whitespace-nowrap">Packed</th>
           <th class="px-1 md:px-6 py-3 whitespace-nowrap">Action</th>
         </tr>
       </thead>
       <TransitionGroup name="checklist" tag="tbody">
+        <tr v-if="items.length === 0">
```

Comment:
Hva er tanken med å utvide listen i endepunktet når knappen bare er tilgjengelig når listen er tom?

Reply:
Tenker det er enten "start fra scratch" eller "kopier fra sist" som er de naturlige valgene, og at jeg slipper håndtere evt. merging av to lister...

Reply:
Uten noen mer forklaring på hva du evt. ønsker anerledes så resolver jeg denne :))